packit_service/worker/handlers/open_scan_hub.py

```diff
@@ -99,17 +100,71 @@ class OpenScanHubTaskFinishedHandler(
     event: OpenScanHubTaskFinishedEvent
     task_name = TaskName.openscanhub_task_finished
 
+    @staticmethod
+    def check_new_findings_identified(url: str) -> bool:
+        """
+        Downloads a JSON file from the specified URL and
+        checks if the 'defects' array is empty.
+
+        Parameters:
+            url (str): The URL of the JSON file.
+
+        Returns:
+            bool: True if the 'defects' array is not empty or we were not
+            able to get it, False otherwise.
+        """
+        logger.info(
+            f"About to check if there were any new findings identified by the scan in {url}."
+        )
+        try:
+            with requests.get(url, timeout=10) as response:
+                response.raise_for_status()
+                data = response.json()
+
+                defects = data.get("defects", [])
+                return len(defects) > 0
```

Comment:
Shall it return number of identified findings and display the number on dashboard? So dashboard would say "The changes in this pull request introduce "x" new findings. Please check the URL for more details.".

Reply:
PTAL